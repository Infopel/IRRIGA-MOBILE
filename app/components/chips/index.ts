export * from './Chips.Item.component'
export * from './chips.component'