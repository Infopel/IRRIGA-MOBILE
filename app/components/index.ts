export * from "../containers/input-field-mapper/input-field-mapper.container"
export * from "./add-formation-participants-model/add-formation-participants-model.component"
export * from "./app-bottom-bar/app-bottom-bar.component"
export * from "./app-toolbar/app-toolbar.component"
export * from "./auto-image/auto-image"
export * from "./beneficiary-info-bottom-sheet/beneficiary-info-bottom-sheet.component"
export * from "./capture/capture.component"
export * from "./chips"
export * from "./comerce-farm-tools-item/farm-tools-list-item.component"
export * from "./comerce-list-item/comerce-list-item.component"
export * from "./coordinates-input/coordinates-input.component"
export * from "./create-comerce-model/create-comerce-model.component"
export * from "./create-earning-model/create-earning-model.component"
export * from "./create-farm-tools-model/create-farm-tools-model.component"
export * from "./create-formation-model/create-formation-model.component"
export * from "./dashboard-search-list-item/dashboard-search-list-item.component"
export * from "./date-input/date-input.component"
export * from "./dropdown-list/dropdown-list.component"
export * from "./earnings-list-item/earnings-list-item.component"
export * from "./form-bottom-bar/form-bottom-bar.component"
export * from "./form-error-message/form-error-message.component"
export * from "./form-row/form-row"
export * from "./form-skeleton/form-skeleton.component"
export * from "./form-toolbar-skeleton/form-toolbar-skeleton.component"
export * from "./form-toolbar/form-toolbar.component"
export * from "./formation-list-item/formation-list-item.component"
export * from "./formation-participants-list-item/formation-participants-list-item.component"
export * from "./gradient-background/gradient-background"
export * from "./multi-select/multi-select.component"
export * from "./regadio-bottom-sheet/regadio-bottom-sheet.component"
export * from "./regadios-list-item/regadios-list-item.component"
export * from "./regadios-list/regadios-list.component"
export * from "./regadios-map/regadios-map.component"
export * from "./screen/screen"
export * from "./search-bar/search-bar.component"

