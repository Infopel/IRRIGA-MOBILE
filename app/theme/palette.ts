export const palette = {
  black: "#1d1d1d",
  white: "#ffffff",
  offWhite: "#e6e6e6",
  orange: "#FBA928",
  orangeDarker: "#EB9918",
  lightGrey: "#939AA4",
  lighterGrey: "#CDD4DA",
  angry: "#dd3333",
  deepPurple: "#5D2555",
}

export const primary = {
  50: "#ddf8ff",
  100: "#b5e5fa",
  200: "#8bd4f2",
  300: "#5fc1eb",
  400: "#35afe3",
  500: "#1c96ca",
  600: "#0c759e",
  700: "#005372",
  800: "#003347",
  900: "#00131c",
}

export const secondary = {
  50: "#deffee",
  100: "#b3fbd3",
  200: "#85f7b8",
  300: "#58f49d",
  400: "#2df182",
  500: "#16d768",
  600: "#0ba750",
  700: "#037838",
  800: "#004820",
  900: "#001a06",
}

export const tertiary = {
  50: "#e2f6ff",
  100: "#c5dfea",
  200: "#a5c9d8",
  300: "#84b3c7",
  400: "#639db5",
  500: "#4a839c",
  600: "#37667a",
  700: "#254959",
  800: "#112d37",
  900: "#001217",
}
