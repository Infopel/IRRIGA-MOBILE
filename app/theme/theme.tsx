import * as React from 'react'
import { extendTheme, NativeBaseProviderProps, NativeBaseProvider as Provider } from "native-base"
import { primary, secondary, tertiary } from "./palette"

const _theme = extendTheme({
  config:{
    useSystemColorMode: true,
    // initialColorMode: 'dark',
  },

  colors: { primary: secondary, secondary: primary, tertiary },
})

export function NativeBaseProvider({theme, ...rest}: NativeBaseProviderProps){
  return <Provider theme={_theme}  {...rest}/>
}
