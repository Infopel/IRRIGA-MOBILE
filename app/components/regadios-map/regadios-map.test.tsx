import * as React from "react"
import { render } from "@testing-library/react-native"
import { RegadiosMap } from "components"
import { NativeBaseProvider } from "theme"
jest.mock('react-native-maps', ()=> 'MapView')
test("should load map", () => {
  const { debug, toJSON } = render(
    <NativeBaseProvider>
      <RegadiosMap items={[]} />
    </NativeBaseProvider>,
  )

  expect(toJSON()).toMatchSnapshot("snap")
})
test("should show list of `regadios`", () => {})
test("should open`regadios` menu", () => {})
