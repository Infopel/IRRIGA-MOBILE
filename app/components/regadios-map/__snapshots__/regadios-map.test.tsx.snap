// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should load map: snap 1`] = `
Array [
  <View
    dataSet={Object {}}
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <MapView
      region={
        Object {
          "latitude": -25.600251,
          "latitudeDelta": 0.015,
          "longitude": 32.237976,
          "longitudeDelta": 0.0121,
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
    <View
      dataSet={Object {}}
      style={
        Object {
          "left": 8,
          "position": "absolute",
          "right": 8,
          "top": 8,
        }
      }
    >
      <View
        dataSet={Object {}}
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
          }
        }
      >
        <View
          dataSet={Object {}}
          isFocused={false}
          style={
            Object {
              "alignItems": "center",
              "borderColor": "#d4d4d4",
              "borderRadius": 4,
              "borderWidth": 1,
              "flex": 1,
              "flexDirection": "row",
              "overflow": "hidden",
            }
          }
        >
          <Text />
          <View
            dataSet={Object {}}
            style={Object {}}
          />
          <TextInput
            accessible={true}
            dataSet={Object {}}
            editable={true}
            onBlur={[Function]}
            onChangeText={[Function]}
            onFocus={[Function]}
            onKeyPress={[Function]}
            placeholder="Nome do regadio"
            placeholderTextColor="#a3a3a3"
            secureTextEntry={false}
            selectionColor="#1f2937"
            style={
              Object {
                "backgroundColor": "transparent",
                "color": "#171717",
                "flex": 1,
                "fontFamily": undefined,
                "fontSize": 12,
                "fontStyle": "normal",
                "fontWeight": "400",
                "height": "100%",
                "paddingBottom": 8,
                "paddingLeft": 12,
                "paddingRight": 12,
                "paddingTop": 8,
                "width": "100%",
              }
            }
          />
        </View>
      </View>
    </View>
  </View>,
  <View
    collapsable={false}
    pointerEvents="box-none"
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "display": "flex",
          "overflow": "hidden",
        },
      ]
    }
  >
    <View
      collapsable={false}
      pointerEvents="none"
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "opacity": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "transform": Array [
            Object {
              "translateY": 0,
            },
            Object {
              "translateX": 0,
            },
            Object {
              "scale": 1,
            },
            Object {
              "scaleX": 1,
            },
            Object {
              "scaleY": 1,
            },
            Object {
              "rotate": "0deg",
            },
          ],
        }
      }
    >
      <View
        accessible={false}
        collapsable={false}
        dataSet={Object {}}
        focusable={true}
        importantForAccessibility="no"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onHoverIn={[Function]}
        onHoverOut={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "rgb(0, 0, 0)",
            "bottom": 0,
            "left": 0,
            "opacity": 0.3,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      />
    </View>
    <View
      collapsable={false}
      pointerEvents="none"
      style={
        Object {
          "bottom": 0,
          "height": "100%",
          "left": 0,
          "opacity": 1,
          "position": "absolute",
          "right": 0,
          "transform": Array [
            Object {
              "translateY": 0,
            },
            Object {
              "translateX": 0,
            },
            Object {
              "scale": 1,
            },
            Object {
              "scaleX": 1,
            },
            Object {
              "scaleY": 1,
            },
            Object {
              "rotate": "0deg",
            },
          ],
        }
      }
    >
      <View
        dataSet={Object {}}
        onLayout={[Function]}
        pointerEvents="box-none"
        style={
          Object {
            "height": "100%",
            "opacity": 0,
          }
        }
      >
        <View
          closeOnOverlayClick={true}
          dataSet={Object {}}
          pointerEvents="box-none"
          style={
            Object {
              "alignItems": "center",
              "height": "100%",
              "justifyContent": "flex-end",
              "width": "100%",
            }
          }
        >
          <View
            collapsable={false}
            onLayout={[Function]}
            pointerEvents="box-none"
            style={
              Object {
                "transform": Array [
                  Object {
                    "translateY": 0,
                  },
                ],
                "width": "100%",
              }
            }
          >
            <View
              collapsable={false}
              dataSet={Object {}}
              onMoveShouldSetResponder={[Function]}
              onMoveShouldSetResponderCapture={[Function]}
              onResponderEnd={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderReject={[Function]}
              onResponderRelease={[Function]}
              onResponderStart={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              onStartShouldSetResponderCapture={[Function]}
              style={
                Object {
                  "paddingBottom": 8,
                  "paddingTop": 8,
                }
              }
            />
            <View
              accessibilityViewIsModal={true}
              aria-modal="true"
              dataSet={Object {}}
              onAccessibilityEscape={[Function]}
              style={
                Object {
                  "alignItems": "flex-start",
                  "backgroundColor": "#fafafa",
                  "borderRadius": 0,
                  "borderTopLeftRadius": 20,
                  "borderTopRightRadius": 20,
                  "elevation": 2,
                  "maxHeight": 1184,
                  "overflow": "hidden",
                  "paddingBottom": 8,
                  "paddingLeft": 64,
                  "paddingRight": 64,
                  "paddingTop": 8,
                  "shadowColor": "#000000",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 0,
                  },
                  "shadowOpacity": 0.2,
                  "shadowRadius": 1.41,
                  "width": "100%",
                }
              }
            >
              <View
                collapsable={false}
                dataSet={Object {}}
                onMoveShouldSetResponder={[Function]}
                onMoveShouldSetResponderCapture={[Function]}
                onResponderEnd={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderReject={[Function]}
                onResponderRelease={[Function]}
                onResponderStart={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                onStartShouldSetResponderCapture={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "marginTop": -8,
                    "paddingBottom": 12,
                    "paddingTop": 12,
                    "width": "100%",
                  }
                }
              >
                <View
                  dataSet={Object {}}
                  style={
                    Object {
                      "backgroundColor": "#737373",
                      "borderRadius": 2,
                      "height": 4,
                      "width": 40,
                    }
                  }
                />
              </View>
              <Text
                dataSet={Object {}}
                style={
                  Object {
                    "backgroundColor": undefined,
                    "color": "#171717",
                    "fontFamily": undefined,
                    "fontSize": null,
                    "fontStyle": "normal",
                    "fontWeight": "700",
                    "letterSpacing": 0,
                    "lineHeight": 30,
                    "textAlign": "left",
                    "textDecorationLine": undefined,
                  }
                }
              />
              <RCTScrollView
                contentContainerStyle={
                  Array [
                    Object {},
                    Object {
                      "dataSet": Object {},
                    },
                  ]
                }
                dataSet={Object {}}
                style={
                  Object {
                    "width": "100%",
                  }
                }
              >
                <View>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 14,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 21,
                        "marginTop": 16,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    Endereço
                  </Text>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 20,
                        "fontStyle": "normal",
                        "fontWeight": "700",
                        "letterSpacing": 0,
                        "lineHeight": 25,
                        "marginTop": 8,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  />
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 14,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 21,
                        "marginTop": 16,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    Área
                  </Text>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 20,
                        "fontStyle": "normal",
                        "fontWeight": "700",
                        "letterSpacing": 0,
                        "lineHeight": 25,
                        "marginTop": 8,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  />
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 14,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 21,
                        "marginTop": 16,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    Beneficiários
                  </Text>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 20,
                        "fontStyle": "normal",
                        "fontWeight": "700",
                        "letterSpacing": 0,
                        "lineHeight": 25,
                        "marginTop": 8,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  />
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 14,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 21,
                        "marginTop": 16,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    Data da última monitoria
                  </Text>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 20,
                        "fontStyle": "normal",
                        "fontWeight": "700",
                        "letterSpacing": 0,
                        "lineHeight": 25,
                        "marginTop": 8,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  />
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 14,
                        "fontStyle": "normal",
                        "fontWeight": "400",
                        "letterSpacing": 0,
                        "lineHeight": 21,
                        "marginTop": 16,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                    Execução física
                  </Text>
                  <Text
                    dataSet={Object {}}
                    style={
                      Object {
                        "backgroundColor": undefined,
                        "color": "#171717",
                        "fontFamily": undefined,
                        "fontSize": 20,
                        "fontStyle": "normal",
                        "fontWeight": "700",
                        "letterSpacing": 0,
                        "lineHeight": 25,
                        "marginTop": 8,
                        "textAlign": "left",
                        "textDecorationLine": undefined,
                      }
                    }
                  >
                     %
                  </Text>
                  <View
                    accessible={true}
                    collapsable={false}
                    dataSet={Object {}}
                    focusable={true}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onHoverIn={[Function]}
                    onHoverOut={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "backgroundColor": "#fafafa",
                        "justifyContent": "flex-start",
                        "paddingBottom": 16,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                        "paddingTop": 16,
                        "width": "100%",
                      }
                    }
                  >
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "flexDirection": "row",
                        }
                      }
                      test={true}
                    >
                      <View
                        dataSet={Object {}}
                        style={Object {}}
                      >
                        <Text
                          dataSet={Object {}}
                          style={
                            Object {
                              "backgroundColor": undefined,
                              "color": "#171717",
                              "fontFamily": undefined,
                              "fontSize": 16,
                              "fontStyle": "normal",
                              "fontWeight": "400",
                              "letterSpacing": 0,
                              "lineHeight": 24,
                              "textDecorationLine": undefined,
                            }
                          }
                        >
                          Carregar fotos
                        </Text>
                      </View>
                    </View>
                  </View>
                  <View
                    accessible={true}
                    collapsable={false}
                    dataSet={Object {}}
                    focusable={true}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onHoverIn={[Function]}
                    onHoverOut={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "backgroundColor": "#fafafa",
                        "justifyContent": "flex-start",
                        "paddingBottom": 16,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                        "paddingTop": 16,
                        "width": "100%",
                      }
                    }
                  >
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "flexDirection": "row",
                        }
                      }
                      test={true}
                    >
                      <View
                        dataSet={Object {}}
                        style={Object {}}
                      >
                        <Text
                          dataSet={Object {}}
                          style={
                            Object {
                              "backgroundColor": undefined,
                              "color": "#171717",
                              "fontFamily": undefined,
                              "fontSize": 16,
                              "fontStyle": "normal",
                              "fontWeight": "400",
                              "letterSpacing": 0,
                              "lineHeight": 24,
                              "textDecorationLine": undefined,
                            }
                          }
                        >
                          Reportar Progresso
                        </Text>
                      </View>
                    </View>
                  </View>
                  <View
                    accessible={true}
                    collapsable={false}
                    dataSet={Object {}}
                    focusable={true}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onHoverIn={[Function]}
                    onHoverOut={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "backgroundColor": "#fafafa",
                        "justifyContent": "flex-start",
                        "paddingBottom": 16,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                        "paddingTop": 16,
                        "width": "100%",
                      }
                    }
                  >
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "flexDirection": "row",
                        }
                      }
                      test={true}
                    >
                      <View
                        dataSet={Object {}}
                        style={Object {}}
                      >
                        <Text
                          dataSet={Object {}}
                          style={
                            Object {
                              "backgroundColor": undefined,
                              "color": "#171717",
                              "fontFamily": undefined,
                              "fontSize": 16,
                              "fontStyle": "normal",
                              "fontWeight": "400",
                              "letterSpacing": 0,
                              "lineHeight": 24,
                              "textDecorationLine": undefined,
                            }
                          }
                        >
                          Localizar no mapa
                        </Text>
                      </View>
                    </View>
                  </View>
                  <View
                    accessible={true}
                    collapsable={false}
                    dataSet={Object {}}
                    focusable={true}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onHoverIn={[Function]}
                    onHoverOut={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "backgroundColor": "#fafafa",
                        "justifyContent": "flex-start",
                        "paddingBottom": 16,
                        "paddingLeft": 16,
                        "paddingRight": 16,
                        "paddingTop": 16,
                        "width": "100%",
                      }
                    }
                  >
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "flexDirection": "row",
                        }
                      }
                      test={true}
                    >
                      <View
                        dataSet={Object {}}
                        style={Object {}}
                      >
                        <Text
                          dataSet={Object {}}
                          style={
                            Object {
                              "backgroundColor": undefined,
                              "color": "#171717",
                              "fontFamily": undefined,
                              "fontSize": 16,
                              "fontStyle": "normal",
                              "fontWeight": "400",
                              "letterSpacing": 0,
                              "lineHeight": 24,
                              "textDecorationLine": undefined,
                            }
                          }
                        >
                          Mais Detalhes
                        </Text>
                      </View>
                    </View>
                  </View>
                </View>
              </RCTScrollView>
            </View>
          </View>
        </View>
      </View>
    </View>
  </View>,
]
`;
